webapi/src/test/scala/org/knora/webapi/e2e/admin/ProjectsADME2ESpec.scala

```diff
@@ -190,10 +165,10 @@ class ProjectsADME2ESpec extends E2ESpec(ProjectsADME2ESpec.config) with Session
                 invalidIri should be(true)
             }
 
-            "return 'BadRequest' if the supplied 'projectIri' is not unique" in {
+            "return 'BadRequest' if the supplied project IRI is not unique" in {
```

Comment:
@subotic the optional `id` of the project is originally tested [here](https://github.com/dasch-swiss/knora-api/pull/1656/files#r450023368) in its PR, in this PR I have refactored it a bit.